### PR TITLE
Add camel-console extension

### LINKED ISSUE #4274: Camel 3.20 - Add camel-console as an extension
We need to have camel-console able to work in camel-quarkus.

### PR DESCRIPTION
**Pull request by zbendhiba**
Fixes #4274

### CODE REVIEW

**Comment by zbendhiba** [comment]
there's something missing I guess, there's no build started

**Comment by zbendhiba** [comment]
I pushed again to kick this, on https://www.githubstatus.com/, we can see there is issue right now with Actions

**Review by aldettinger — COMMENTED** [review]
Should we open follow up issues to:
 + auto-enable the dev console when the extension is used ?
 + improve test coverage ?

**Comment by zbendhiba** [comment]
> * improve test coverage ?
yes. I'd work on that to determine what's needed for dev console.

> * auto-enable the dev console when the extension is used ?
Auto-enable? I guess is working on dev console. The dev console is always enabled in dev mode. the issue exists already https://github.com/apache/camel-quarkus/issues/2156

**Review by aldettinger — APPROVED** [review]

**Comment by zbendhiba** [comment]
I think the failure is unrelated to this; Maybe  camel-quarkus-integration-test-syslog a flaky test on windows ??

**Review by davsclaus — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

File: integration-tests-jvm/console/pom.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            <activation>
+                <property>
+                    <name>!noVirtualDependencies</name>
+                </property>
+            </activation>
+            <dependencies>
+                <!-- The following dependencies guarantee that this module is built after them. You can update them by running `mvn process-resources -Pformat -N` from the source tree root directory -->
+                <dependency>
+                    <groupId>org.apache.camel.quarkus</groupId>
+                    <artifactId>camel-quarkus-cometd-deployment</artifactId>
```
Comment (jamesnetherton) [inline integration-tests-jvm/console/pom.xml:70]: I think we want `camel-quarkus-console-deployment`.

Comment (zbendhiba) [reply integration-tests-jvm/console/pom.xml]: @jamesnetherton Thanks so much for catching this. I pushed a fix.

**Comment by jamesnetherton** [comment]
Seems we're running out of heap space. Maybe try increasing this to `4500m`:

https://github.com/apache/camel-quarkus/blob/main/.github/workflows/ci-build.yaml#L65

**Comment by essobedo** [comment]
@jamesnetherton FYI I've just tried to increase it to `4400m` within the context of https://github.com/apache/camel-quarkus/pull/4414, let's see the results

**Comment by zbendhiba** [comment]
Let me rebase then

**Comment by zbendhiba** [comment]
> Let me rebase then

I added a commit to increase it, let's see if it works, and make sure to have it merged only once :)

**Comment by essobedo** [comment]
I confirmed that it worked with `4400m`

**Comment by davsclaus** [comment]
yes this is good to get included, and this will be needed for camel jbang with quarkus also.

**Comment by aldettinger** [comment]
@zbendhiba Do you have time to solve the conflict ? Rebasing on main should be enough.

**Comment by zbendhiba** [comment]
Sûre. Let me rebase and regenerate the flattened boys

**Comment by zbendhiba** [comment]
there's the need to delete the commit I did to increase heap space because it's already merged in the other PR.

**Comment by essobedo** [comment]
Yeah, you will need to do a hard reset on `4ea7727c0f5b182765ac73f48658c5ac40e0cf9e` (`git reset --hard 4ea7727c0f5b182765ac73f48658c5ac40e0cf9e`)

**Pull request merged**